packages/paste-core/primitives/box/src/types.ts

```diff
@@ -8,36 +8,16 @@ import type {
   SpaceProps,
   TypographyProps,
 } from '@twilio-paste/style-props';
+import type {TLengthStyledSystem} from '@twilio-paste/styling-library';
 import type {CustomTheme} from '@twilio-paste/customization';
-import type {
-  AnimationProperty,
-  AppearanceProperty,
-  BorderCollapseProperty,
-  BorderSpacingProperty,
-  BoxSizingProperty,
-  ClipProperty,
-  CursorProperty,
-  FloatProperty,
-  FontVariantNumericProperty,
-  ListStyleImageProperty,
-  ListStylePositionProperty,
-  ListStyleTypeProperty,
-  ObjectFitProperty,
-  ObjectPositionProperty,
-  OpacityProperty,
-  OutlineProperty,
-  PointerEventsProperty,
-  ResizeProperty,
-  TableLayoutProperty,
-  TransformOriginProperty,
-  TransformProperty,
-  TransitionProperty,
-  UserSelectProperty,
-  VisibilityProperty,
-  WillChangeProperty,
-} from 'csstype';
+import type {StandardLonghandProperties, StandardShorthandProperties} from 'csstype';
+
+// 'csstype';
 import type {PseudoPropStyles} from './PseudoPropStyles';
 
+type LonghandProperties = StandardLonghandProperties<TLengthStyledSystem>;
```

Comment:
✂️ This needs to be edited. If [the gatsby upgrade](https://github.com/twilio-labs/paste/pull/2233) is merged first, I can back out all of these changes.